client/my-sites/post-type-list/index.jsx

```diff
@@ -46,6 +46,7 @@ class PostTypeList extends Component {
 		super( ...arguments );
 
 		this.renderPostRow = this.renderPostRow.bind( this );
+		this.cellRendererWrapper = this.cellRendererWrapper.bind( this );
```

Comment:
You're right about `renderPostRow` though, that one no longer needs binding, though it did need it before. :)